### ISSUE TITLE
trefftzComputation is defined twice

### ISSUE THREAD

**Issue by ArneBachmann** [DLR-SL/CPACS#481]
One time as an anonymous type definition inside `aeroFuseToolParametersType` and one time inside `vSAeroToolParametersType`, but both are defined as
```
<xsd:complexType>
  <xsd:simpleContent>
    <xsd:restriction base="stringBaseType">
      <xsd:enumeration value="none"/>
      <xsd:enumeration value="cfx"/>
      <xsd:enumeration value="cfz"/>
      <xsd:enumeration value="cfxcfz"/>
    </xsd:restriction>
  </xsd:simpleContent>
</xsd:complexType>
```

**Comment by ArneBachmann** [comment]
Let's wait for the toolspecific-as-own-namespaces discussion and decide then if to remove from CPACS or combine both definitions instead.

**Comment by MarAlder** [comment]
closed by  #611
